openbas-api/src/main/java/io/openbas/utils/InjectMapper.java

```diff
@@ -97,23 +96,92 @@ private PayloadSimple toPayloadSimple(Optional<Payload> payload) {
   }
 
   // -- STATUS to STATUSIMPLE --
+  private <T extends InjectStatusOutput> T buildInjectStatusOutput(
+      T output, BaseInjectStatus status, List<ExecutionTraces> executionTraces) {
+    output.setId(status.getId());
+    output.setName(Optional.ofNullable(status.getName()).map(ExecutionStatus::name).orElse(null));
```

Comment:
not a fan of using optionals here when a simple ternary expression would do the job just fine.
`output.setName(status.getName() == null ? null : status.getName().getName())`
Your choice :)